images/salt-master/Dockerfile

```
@@ -1,32 +1,38 @@
-FROM centos:7.6.1810
+# SHA256 digest of the base image
+ARG BASE_IMAGE_SHA256
+ARG BASE_IMAGE=docker.io/rockylinux
 
-MAINTAINER moonshot-platform <moonshot-platform@scality.com>
+FROM ${BASE_IMAGE}@sha256:${BASE_IMAGE_SHA256}
 
 # Versions to use
 ARG SALT_VERSION
-ENV LC_ALL=en_US.UTF-8
 
-# Install saltstack
+# Install Saltstack and other dependencies
 RUN printf "[saltstack-repo]\n\
 name=SaltStack repo for RHEL/CentOS \$releasever\n\
 baseurl=https://repo.saltstack.com/py3/redhat/\$releasever/\$basearch/archive/%s\n\
 enabled=1\n\
 gpgcheck=1\n\
 gpgkey=https://repo.saltstack.com/py3/redhat/\$releasever/\$basearch/archive/%s/SALTSTACK-GPG-KEY.pub\n" ${SALT_VERSION} ${SALT_VERSION} >/etc/yum.repos.d/saltstack.repo \
- && rpm --import https://repo.saltstack.com/py3/redhat/7/x86_64/archive/${SALT_VERSION}/SALTSTACK-GPG-KEY.pub \
- && yum clean expire-cache \
- && yum install -y epel-release \
- && yum install -y python3-pip \
+ && rpm --import https://repo.saltstack.com/py3/redhat/8/x86_64/archive/${SALT_VERSION}/SALTSTACK-GPG-KEY.pub \
+ && dnf clean expire-cache \
+ && dnf update -y \
+ && dnf install -y glibc-all-langpacks langpacks-en \
+ && dnf install -y epel-release \
+ && dnf install -y python3-pip \
  && pip3 install pip==20.1 \
  && pip3 install "etcd3 != 0.11.0" \
```

nitpick: It's not consistent :cry: 
```suggestion
 && pip3 install "pip == 20.1" \
 && pip3 install "etcd3 != 0.11.0" \
```